src/client/pages/EmailSent.tsx

```diff
@@ -56,7 +56,7 @@ export const EmailSent = ({
       )}
       <MainBodyText>Please follow the instructions in this email.</MainBodyText>
       <MainBodyText>
-        <b>The link is only valid for 30 minutes.</b>
+        <b>The link is only valid for 60 minutes.</b>
```

Comment:
this seems very repetitive. Maybe in the future we can extract this to some common location, either the expiration description or the amount of minutes (if we are sure we are never going to express it in hours or days)